RecoLocalMuon/MuonSysAging/python/CSCChamberMasker_cfi.py

```diff
@@ -0,0 +1,11 @@
+
+import FWCore.ParameterSet.Config as cms
+
+CSCChamberMasker = cms.EDProducer('CSCChamberMasker',
```

Comment:
module instances should follow the same CMSSW style as C++ code and use lower case

More importantly, I think that these files should be removed and corresponding defaults created by the framework as a part of fillDescriptions implementation.